### PR TITLE
Fix Python requirments sync of workflows

### PR DESCRIPTION
**Pull request by BjoernAtBosch**
## Describe your changes

Fix for commit "Use Python requirements.txt for CI workflow also (#38)":
SDK - Build Base Container workflow was not regarded

## Issue ticket number and link

none

## Checklist - Manual tasks

* [ ] Examples are executing successfully
* [ ] Created/updated unit tests. Code Coverage percentage on new code shall be >= 80%.
* [ ] Created/updated integration tests.
* [ ] Devcontainer can be opened successfully
* [ ] Devcontainer can be opened successfully behind a corporate proxy
* [ ] Devcontainer can be re-built successfully
* [ ] Extended the documentation (e.g. README.md, CONTRIBUTING.md, Velocitas)

### CODE REVIEW

File: sdk/Dockerfile [hunk 1]
```diff
@@ -26,7 +26,7 @@ RUN apk update && \
     apk add linux-headers && \ 
     apk add perl && \
     apk add bash && \
-    pip3 install conan
+    pip3 install -r requirements.txt
```
Comment (doosuu) [inline sdk/Dockerfile:29]: Won't work, file is not available in container context, yet

Comment (BjoernAtBosch) [reply sdk/Dockerfile]: Reverted back to "simple" fix - we need to check anyway if this container is still needed.
And lifecycle mgm needs also being introduced

File: .github/workflows/ci.yml [hunk 2]
```diff
@@ -1,4 +1,4 @@
-# Copyright (c) 2022 Robert Bosch GmbH
+# Copyright (c) 2022-2023 Robert Bosch GmbH
```
Comment (BjoernAtBosch) [inline .github/workflows/ci.yml:1]: Was changed in last PR ...

**Review by MP91 — APPROVED** [review]
LGTM

**Pull request merged**